components/AddCatCard.tsx

```diff
@@ -0,0 +1,329 @@
+import { capitalize } from 'lodash';
+import React, { useState } from 'react';
+import { Image, StyleSheet, View } from 'react-native';
+import { Button, Caption, Card, TextInput, useTheme } from 'react-native-paper';
+import {
+  DatePickerInput,
+  en,
+  registerTranslation,
+} from 'react-native-paper-dates';
+import SelectDropdown from 'react-native-select-dropdown';
+import imagePlaceholder from '../assets/image-placeholder.png';
+import { Cat } from '../models/Cat';
+import { User } from '../models/User';
+import { InputField } from './InputField';
+
+registerTranslation('en', en);
```

Comment:
why do we need this?

Reply:
Otherwise a warning appears: `[react-native-paper-dates] en is not registered, key: next
at node_modules/react-native-paper-dates/src/translations/utils.ts:28:4 in getTranslation
at node_modules/react-native-paper-dates/src/Date/CalendarHeader.tsx:65:34 in CalendarHeader`

Reply:
ok